test/extended/operators/olm.go

```diff
@@ -105,4 +106,26 @@ var _ = g.Describe("[Feature:Platform] OLM should", func() {
 		}
 	})
 
+	// OCP-23395 - Deleted catalog registry pods and verify if them are recreated automatically
+	// author: bandrade@redhat.com
+	g.It("Recreate catalog registry pods and if them are deleted", func() {
+		msg, err := oc.SetNamespace("openshift-operator-lifecycle-manager").AsAdmin().Run("delete").Args("pods", "-l app=catalog-operator").Output()
```

Comment:
The `catalog-operator` is not a registry pod, but the operator itself. There are pre-existing registry pods in the `openshift-operators` namespace that you can use for this test.

Reply:
changed as suggested